Source/Blazorise/Components/Modal/Modal.razor.cs

```diff
@@ -61,6 +61,32 @@ public partial class Modal : BaseComponent, ICloseActivator
 
         #region Methods
 
+        /// <inheritdoc/>
+        public override async Task SetParametersAsync( ParameterView parameters )
+        {
+            if ( parameters.TryGetValue<bool>( nameof( Visible ), out var visibleResult ) && state.Visible != visibleResult )
+            {
+                if ( visibleResult == true )
+                {
+                    state = state with { Visible = true };
+
+                    HandleVisibilityStyles( true );
+                    RaiseEvents( true );
+                }
+                else if ( visibleResult == false && await IsSafeToClose() )
+                {
+                    state = state with { Visible = false };
+
+                    HandleVisibilityStyles( false );
+                    RaiseEvents( false );
+                }
```

Comment:
If these are always done in 'bulk' maybe you can extract these into a method with a bool argument?